client/src/debug/FabricGoDebugConfigurationProvider.ts

```diff
@@ -29,9 +29,10 @@ export class FabricGoDebugConfigurationProvider extends FabricDebugConfiguration
         ];
     }
 
-    protected async getChaincodeName(): Promise<string> {
+    protected async getChaincodeNameAndVersion(): Promise<{name: string, version: string}> {
         const name: string = await UserInputUtil.showInputBox('Enter a name for your Go package'); // Getting the specified name and package from the user
-        return name;
+        const version: string = await UserInputUtil.showInputBox('Enter a version for your Go package');
```

Comment:
Should we not be checking if the user has cancelled giving us a name or version?

Reply:
this is checked in the FabricDebugConfigurationProvider